docs/examples/201/wvd-create-hostpool/modules/managedDisks-customimagevm.bicep

```diff
@@ -182,7 +182,7 @@ resource vm 'Microsoft.Compute/virtualMachines@2018-10-01' = [for i in range(0,
     hardwareProfile: {
       vmSize: rdshVmSize
     }
-    availabilitySet: ((availabilityOption == 'AvailabilitySet') ? vmAvailabilitySetResourceId : json('null'))
+    availabilitySet: ((availabilityOption == 'AvailabilitySet') ? vmAvailabilitySetResourceId : any(null))
```

Comment:
>any [](http://example.com/codeflow?start=96&length=3)

Is `any` still needed? Any property not marked as required should allow null at this point.

Reply:
Good point - fixed where possible